ksqldb-parser/src/main/java/io/confluent/ksql/parser/tree/CreateTable.java

```diff
@@ -27,15 +27,22 @@
 
 @Immutable
 public class CreateTable extends CreateSource implements ExecutableDdlStatement {
+  public enum Type {
+    SOURCE,
+    NORMAL
+  }
```

Comment:
I had to create the `Type` inside the `CreateTable.Type` enum because that gives better meaning to the table type we want to use (source or normal). I could have added to the parent class, `CreateSource.Type`, but that sounds like source is a table or stream type. I'm still not sure whether I put the type in the right place, so feedback is welcome.

Reply:
Some ideas:
* Put it under `CreateSource` with a different name, like `ownershipType`
* Add the enum to `KsqlConstants` in `ksqldb-common` - `CreateTableCommand` won't have to use strings anymore